ftw/file/locales/en/LC_MESSAGES/ftw.file.po

```diff
@@ -93,6 +93,11 @@ msgstr ""
 msgid "download"
 msgstr ""
 
+#. Default: "Download not possible because the file contains a virus (${name})."
+#: ./ftw/file/fields.py:99
+msgid "download_not_possible"
+msgstr ""
```

Comment:
Download blocked. The malware ${name} has been found in the file.

Reply:
ok, done